bump.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
you could also add `bump` as a `devDependency` so se can use it as an `npm` task and then no one needs to install it as a global package

Reply:
I remember adding it, don't know how it slipped, will do that now 👍